korangar/src/world/cameras/player.rs

```diff
@@ -57,6 +58,10 @@ impl PlayerCamera {
         self.view_angle.move_desired(rotation * ROTATION_SPEED);
     }
 
+    pub fn reset_rotate(&mut self) {
```

Comment:
```suggestion
    pub fn reset_rotation (&mut self) {
```

I think reset_rotation would be a better name.